solidity/dashboard/src/components/FormCheckbox.jsx

```diff
@@ -7,10 +7,10 @@ const FormCheckbox = ({ label, withError = false, ...props }) => {
 
   return (
     <>
-      <label>
-        <input checked={field.value} {...field} {...props} />
-        &nbsp;{label}
-      </label>
+      <div className="form-checkbox">
+        <input id={props.name} checked={field.value} {...field} {...props} />
+        <label htmlFor={props.name}>{label}</label>
+      </div>
```

Comment:
I agree the `text` cursor seems off. I believe it should be `cursor: default` instead of `pointer` according to [docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox).

Reply:
Please see 80e9a10